sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/implementation/util/BlobSasImplUtil.java

```diff
@@ -97,7 +104,25 @@ public BlobSasImplUtil(BlobServiceSasSignatureValues sasValues, String container
      */
     public BlobSasImplUtil(BlobServiceSasSignatureValues sasValues, String containerName, String blobName,
         String snapshotId) {
+        this(sasValues, containerName, blobName, snapshotId, null);
+    }
+
+    /**
+     * Creates a new {@link BlobSasImplUtil} with the specified parameters
+     *
+     * @param sasValues {@link BlobServiceSasSignatureValues}
+     * @param containerName The container name
+     * @param blobName The blob name
+     * @param snapshotId The snapshot id
+     * @param versionId The version id
+     */
+    public BlobSasImplUtil(BlobServiceSasSignatureValues sasValues, String containerName, String blobName,
```

Comment:
This code is in `implementation` so feel free to update the old API as it is non-breaking change.

Reply:
Great! Done.